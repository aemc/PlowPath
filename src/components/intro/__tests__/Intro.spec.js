import React from "react";
import Component from "../Intro";
import { shallow } from "enzyme";

const defaultProps = {
  classes: {}
};

describe("Intro", () => {
  it("renders", () => {
    const wrapper = shallow(<Component {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
