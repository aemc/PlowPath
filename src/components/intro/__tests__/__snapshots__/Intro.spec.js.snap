// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Intro)
    classes={Object {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "container": "Intro-container-8",
        "img": "Intro-img-5",
        "legendimg": "Intro-legendimg-6",
        "legendsmall": "Intro-legendsmall-7",
        "legendtext": "Intro-legendtext-4",
        "root": "Intro-root-1",
        "text1": "Intro-text1-2",
        "text2": "Intro-text2-3",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "container": "Intro-container-8",
          "img": "Intro-img-5",
          "legendimg": "Intro-legendimg-6",
          "legendsmall": "Intro-legendsmall-7",
          "legendtext": "Intro-legendtext-4",
          "root": "Intro-root-1",
          "text1": "Intro-text1-2",
          "text2": "Intro-text2-3",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
